instance/tests/api/test_openedx_appserver.py

```diff
@@ -145,7 +145,7 @@ def test_get_details(self):
         self.assertIn(('public_ip', None), server_data)
         # The API call will try to start the server, which will fail, since we're
         # not actually talking to an OpenStack instance when unit tests are running
-        self.assertIn(('status', 'failed'), server_data)
+        self.assertIn(('status', 'unknown'), server_data)
```

Comment:
I'm not clear on what caused this to change, or why the tests below no longer get "Failed to start server: Not found (HTTP 404)" when attempting to start the server?